website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -63,30 +65,39 @@ We recommend providing the agency or freelancers with the following guidelines:
 
 Obviously, a big conference like KubeCon has more resources than a local community event — we get that. Since we also want to be included in smaller events, we’ll offer various options. To be clear, the more cost-effective options aren’t as inclusive, meaning we’ll miss out on many opportunities our hearing peers benefit from. Opt for the more cost-effective route only if there is no alternative that provides accessibility. We deserve to be included and appreciate your support.
 
-Sign language interpreters are the most valuable but also the most expensive accommodation. Incorporate the cost into the overall conference cost. It should be no different than including audio/video equipment, swag, food, refreshments, etc. into your budget.
+For deaf signing members, sign language interpreters are the most valuable but also the most expensive accommodation. Incorporate the cost into the overall conference cost. It should be no different than including audio/video equipment, swag, food, refreshments, etc. into your budget.
+
+**Note:** The Linux Foundation is building its own team of freelance interpreters. If you are looking for interpreters for your tech conferences, let us know, and we can make recommendations. 
 
 #### Include interpreters in recording
 
 If you record your sessions, be sure the interpreters are on screen so deaf attendees can benefit from your accessibility effort even when watching them on-demand.
 
 ### Captions
 
-There is no need to provide interpreters for every session, but captions should be available for all sessions. While there are lots of live captions software options, not all work well. Using a tool with low accuracy can be very confusing for deaf/hoh individuals, so select one that has been proven to work effectively.
+There is no need to provide interpreters for every session, but captions should be available for all sessions. While there are many live caption software options, not all work well. Using a tool with low accuracy can be very confusing for deaf/hoh individuals, so select one that has been proven to work effectively.
 
-Some caption tools work for virtual events only, others for in-person and virtual. 
+Some caption tools work only for virtual events, others for in-person and virtual events.
 
 #### Some considerations
 
 Captions at the bottom of the screen are not visible unless attendees are sitting at the front. Consider adding them at the top. If that isn't feasible,  you might want to place the screen higher to ensure people in the back can also read them. 
 
+<img width="906" alt="Captions on top vs bottom of screen" src="https://github.com/cncf/tag-contributor-strategy/assets/74001907/b8a08ec0-162b-4465-84d5-a082ed078154">
```

Comment:
@sandeepkanabar, is there a way to cap how much the image scales up? It doesn't need to be as big, and it gets bigger and bigger as you increase your window.